### ISSUE TITLE
Latest Uberon release not highlighted

### ISSUE THREAD

**Issue by paola-scibite** [obophenotype/uberon#1656]
Hi,

The [Uberon entry in OLS](https://www.ebi.ac.uk/ols/ontologies/uberon) says that the latest version is from 2020-09-16. Judging from the [list of commits](https://github.com/obophenotype/uberon/commits/master) since the previous version (Nov 2019), it looks like the current versions contains interesting new terms as well as more technical changes. However, there are no release notes on GitHub, and the latest release is not listed in the [Uberon portal](http://uberon.github.io/) (note that the portal is referred to as Uberon's Homepage/Home in the [OBO Foundry](http://www.obofoundry.org/ontology/uberon.html) and in [Ontobee](http://www.ontobee.org/ontology/UBERON)). The latest release is indeed in [BioPortal](https://bioportal.bioontology.org/ontologies/UBERON/?p=summary), though the version info is incorrect due to the OWL property being examined (see https://github.com/obophenotype/uberon/issues/1617). 

In summary,
Release notes would be helpful (especially as Uberon doesn't release often).
Should the [Uberon portal](http://uberon.github.io/) be discontinued and replaced by this GitHub tracker in all its external references?
Would Uberon please consider including a versionInfo property along with versionIRI to address the reporting issue in BioPortal (and potentially elsewhere). 

Thanks!
Paola
